packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -0,0 +1,125 @@
+@import '../../../global/global.pcss';
+
+[part='container'] {
+  box-shadow: 0px 4px 8px rgba(4, 8, 31, 0.08), inset 0px 0px 0px 1px #e5e8e8;
+  margin: 0 auto 30px auto;
+}
+
+[part='header-label'] {
+  border-radius: 8px;
+  color: #1372ec;
+  padding: 8px 12px;
+  font-weight: 500;
+  display: inline-block;
+}
+
+[part='generated-text'] {
+  margin-bottom: 0;
+  margin-top: 20px;
+  color: var(--atomic-neutral-dark);
+  line-height: 26px;
+  white-space: pre-wrap;
+}
+
+.header {
+  align-items: center;
+  justify-content: center;
+}
+
+.feedback-buttons {
+  margin-left: auto;
+  gap: 0.5rem;
+
+  .feedback-button {
+    padding: 8px;
+    width: 36px;
+    height: 36px;
+    border-radius: 8px;
+
+    &.active {
+      &.like {
+        background: #c5f4e6;
+      }
+
+      &.dislike {
+        background: #f6a5a8;
+      }
+    }
+  }
+}
+
+.text-bg-blue {
+  background: #edf6ff;
+  color: #1372ec;
```

Comment:
Same thing here, we should make sure to use variables for colors.

You may need to add new colors to `coveo.css` and `accessible.css` if we don't have similar ones to what you need.